lib/MediaStreamPlayer.tsx

```diff
@@ -21,33 +37,78 @@ export class MediaStreamPlayer extends HTMLElement {
     this.setAttribute('hostname', value)
   }
 
+  get autoplay() {
+    return this.getAttribute('autoplay') ?? 'false'
+  }
+
+  set autoplay(value: string) {
+    this.setAttribute('autoplay', value)
+  }
+
```

Comment:
We should mimic the real autoplay attribute, i.e. if it is missing it's false, otherwise true. Following an example for the `disabled` property (https://developers.google.com/web/fundamentals/web-components/customelements) we would getL
```
// A getter/setter for a disabled property.
  get autoplay() {
    return this.hasAttribute('autoplay');
  }

  set autoplay(val) {
    // Reflect the value of the autoplay property as an HTML attribute.
    if (val) {
      this.setAttribute('autoplay', '');
    } else {
      this.removeAttribute('autoplay');
    }
  }
```